### DISCUSSION TITLE
Control Panel: Clear Play History

### DISCUSSION THREAD

**Discussion by kdecks** [RetroAchievements/RAWeb#659] [Ideas]
Commonly requested feature. Right now play history shows 15 most recent changes.

Some players would like to be able to clear their play history for whatever reason. This shouldn't be too difficult, and I don't see much drawback.

**Comment by televandalist** [comment]
Yes, I look forward to not looking like a complete psycho. 

![w](https://user-images.githubusercontent.com/45054151/110203727-618e1500-7e35-11eb-86c7-48369f929b7c.PNG)

**Comment by meleu** [comment]
I don't think it would be simple to just "clear" the play history (meaning that it would start a brand new play history).

An alternative would be "do not show Play History".

---

* my definition of "simple": no need to change the Data Base.

**Comment by televandalist** [comment]
> I don't think it would be simple to just "clear" the play history (meaning that it would start a brand new play history).
> 
> An alternative would be "do not show Play History".
> 
> * my definition of "simple": no need to change the Data Base.

What about removing games individually if there's no achievements earned in them?
